src/lib/Behat/BusinessContext/ContentUpdateContext.php

```diff
@@ -28,6 +30,15 @@ public function iSetFields(TableNode $table): void
         }
     }
 
+    /**
+     * @When I select :contentPath from Image Asset Repository for :fieldName field
+     */
+    public function selectContentFromIARepository(string $contentPath, string $fieldname): void
+    {
+        $imas = ElementFactory::createElement($this->browserContext, ContentUpdateForm::ELEMENT_NAME);
```

Comment:
`$imas` is not a meaningful variable name. In this case it should be something like `$contentUpdateForm`.